xml/gl.xml

```diff
@@ -26206,7 +26201,6 @@ typedef unsigned int GLhandleARB;
             <param><ptype>GLenum</ptype> <name>format</name></param>
             <param><ptype>GLenum</ptype> <name>type</name></param>
             <param len="COMPSIZE(format,type,width,height,depth)">const void *<name>pixels</name></param>
-            <alias name="glTexSubImage3D"/>
```

Comment:
OK, I reported this to github and they said "I've reported this as a bug to our engineering team, though I can't give an ETA on when this will be fixed.". Woohoo. In the future I may ask people to attach a locally-generated 'git diff' for context, if something appears to be bogus. Accepting this.